### PR TITLE
Add module to set global admin values

### PR DESCRIPTION
**Pull request by sdodsley**
##### SUMMARY
New module to set max_logins, lockout_duration and min password length globally.
Requires REST 2.3, aka Purity//FB 3.3.0

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
purefb_admin.py

### CODE REVIEW

File: collections/ansible_collections/purestorage/flashblade/changelogs/fragments/164_add_admin.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - purefa_admin - New module to manage global admin settings
```
Comment (rdeenik) [inline collections/ansible_collections/purestorage/flashblade/changelogs/fragments/164_add_admin.yaml:2]: Should be `purefb_admin`

File: collections/ansible_collections/purestorage/flashblade/changelogs/fragments/176_nfs_export_policies.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+minor_changes:
+ - purefb_policy - Add NFS export policies, with rules, as a new policy type
+ - purefb_info - Add NFS export policies and rules
```
Comment (rdeenik) [inline collections/ansible_collections/purestorage/flashblade/changelogs/fragments/176_nfs_export_policies.yaml:3]: Not sure if these changes are related to this PR?

**Review by rdeenik — CHANGES_REQUESTED** [review]
Is there a way to remove these settings as well? The GUI does allows you to unset.

**Comment by sdodsley** [comment]
> Is there a way to remove these settings as well? The GUI does allows you to unset.

Setting the values to zero will remove them

**Review by rdeenik — APPROVED** [review]
Let's ship this. Quick note the values cannot be removed, as 0 is not an accepted value, but this seems related to the API.

**Pull request merged**